tests/unit/test_network_connections.py

```diff
@@ -15,7 +16,10 @@
 
 sys.modules["ansible.module_utils.basic"] = mock.Mock()
 
-# pylint: disable=import-error, wrong-import-position
+try:
+    import network_lsr
+except ModuleNotFoundError:
```

Comment:
looks like this breaks py27 support - `ModuleNotFoundError` not supported - why not use `ImportError`?

Reply:
done.